elephant/signal_proc.py

```diff
@@ -0,0 +1,130 @@
+'''
+signal_proc module
```

Comment:
"signal_processing" is fine with me as well.
I will fix the module name in my pull request for the butterworth function accordingly.